plugins/modules/postgresql_pg_hba.py

```diff
@@ -1227,13 +1276,16 @@ def main():
         dest=dict(type='path', required=True),
         method=dict(type='str', default='md5', choices=PG_HBA_METHODS),
         netmask=dict(type='str'),
+        # TODO this can probably be changed to dict without breaking
         options=dict(type='str'),
+        # DEPRECATED, does nothing
```

Comment:
```suggestion
        # DEPRECATED, does nothing
```
should it be deprecated deprecated? please see https://docs.ansible.com/ansible/latest/dev_guide/developing_program_flow_modules.html#argument-spec , there are corresponding fields similar to as we used when deprecating aliases

Reply:
This actually doesn't do anything anymore. The module behaves as if this is always `true` (which doesn't break anything). So this is only here to ensure the module doesn't throw an error when this is specified. I added the additional fields.

Reply:
@toydarian ok, thanks for explaining, I'll create an issue to properly deprecate it in another PR and remove it as we shouldn't keep any arguments that do nothing